include/erlcloud_ddb_streams.hrl

```diff
@@ -36,8 +36,24 @@
 -record(ddb_streams_describe_stream,
         {stream_description :: #ddb_streams_stream_description{}
         }).
+-record(ddb_streams_record_dynamodb,
```

Comment:
I think that name actually corresponds to the `#ddb_streams_record{}` below, which ought perhaps to be called  `#ddb_streams_stream_record{}`. This record ought in turn to be called  `#ddb_streams_stream_record_dynamodb{}` or something. I left out the singular `stream` prefix in my pattern.